### PR TITLE
Fixes for FBC V3

### LINKED ISSUE #390: fbc-v3: Usage practices for key/value pairs
This is a collection issue for best practices etc on how to use key/value pairs in fbc-v3.
Please post anything that comes up to mind on what can/should and what should not be encoded in key/value pairs. And if you say something shouldn't indicate how it should be encoded.

I'm starting with a view things that should "not" be annotated in key/value pairs:
- layouting information -> use the layout extension
- GPR information -> use Geneproducts etc. 
- ...

### LINKED ISSUE #394: fbc-v3: Add orcid to authors, remove thanking of authors
Some smaller things noticed by @draeger.
- it would be great if we could add orcid ids to the authors on the specification. The emails will stop working at some point but orcid ids are more persistent. Also the specification would be linked to the orcid entry
- check if Frank's affiliation at CalTech is correct
- on page 42 we thank some of the authors. We should not thank the authors, but only people not on the author list.

### LINKED ISSUE #398: fbc v3: constraint example could be clearer with semantic IDs
The example user constraint takes a lot of reference-following to understand, and none of the references *mean* anything, so far as I can tell.  Here's the current example:

```xml
<listOfParameters>
 <parameter id="uc1" value="5" constant="True"/>
 <parameter id="uc2lb" value="2" constant="True"/>
 <parameter id="uc2ub" value="INF" constant="True"/>
 <parameter id="ucco1a" value="1" constant="True"/>
 <parameter id="ucco1b" value="-1" constant="True"/>
 <parameter id="ucco2a" value="2" constant="True"/>
 <parameter id="ucco2b" value="-1" constant="True"/>
 <parameter id="p1var" value="NaN" constant="False"/>
</listOfParameters>

<fbc:listOfUserConstraints>
 <fbc:userContraint fbc:id="uc1" fbc:lowerBound="uc1" fbc:upperBound="uc1">
  <fbc:listOfUserConstraintComponents>
   <fbc:userConstraintComponent fbc:coefficient="ucco1a" fbc:variable="RGLX"
    variableType="linear"/>
   <fbc:userConstraintComponent fbc:coefficient="ucco1b" fbc:variable="RBTK"
    variableType="linear"/>
  </fbc:listOfUserConstraintComponents>
 </fbc:userContraint>
 <fbc:userContraint fbc:id="uc2" fbc:lowerBound="uc2lb" fbc:upperBound="uc2ub">
  <fbc:listOfUserConstraintComponents>
   <fbc:userConstraintComponent fbc:coefficient="ucco2a" fbc:variable="p1var"
    variableType="linear"/>
   <fbc:userConstraintComponent fbc:coefficient="ucco2b" fbc:variable="RGDP"
    variableType="linear"/>
  </fbc:listOfUserConstraintComponents>
 </fbc:userContraint>
</fbc:listOfUserConstraints>
```

I would like the following, instead:

```xml
<listOfParameters>
 <parameter id="inf" value="INF" constant="True"/>
 <parameter id="five" value="5" constant="True"/>
 <parameter id="two" value="2" constant="True"/>
 <parameter id="one" value="1" constant="True"/>
 <parameter id="negone" value="-1" constant="True"/>
 <parameter id="p1var" constant="False"/>
</listOfParameters>

<fbc:listOfUserConstraints>
 <fbc:userContraint fbc:id="uc1" fbc:lowerBound="five" fbc:upperBound="five">
  <fbc:listOfUserConstraintComponents>
   <fbc:userConstraintComponent fbc:coefficient="one" fbc:variable="RGLX"
    variableType="linear"/>
   <fbc:userConstraintComponent fbc:coefficient="negone" fbc:variable="RBTK"
    variableType="linear"/>
  </fbc:listOfUserConstraintComponents>
 </fbc:userContraint>
 <fbc:userContraint fbc:id="uc2" fbc:lowerBound="two" fbc:upperBound="inf">
  <fbc:listOfUserConstraintComponents>
   <fbc:userConstraintComponent fbc:coefficient="two" fbc:variable="p1var"
    variableType="linear"/>
   <fbc:userConstraintComponent fbc:coefficient="negone" fbc:variable="RGDP"
    variableType="linear"/>
  </fbc:listOfUserConstraintComponents>
 </fbc:userContraint>
</fbc:listOfUserConstraints>
```
(Also, I removed the 'NaN' value for p1var, assuming it was set elsewhere, and NaN is confusing in this context.)

### LINKED ISSUE #400: fbc v3: DocumentConventions don't say why things are red
Section 1.4 should say that red bits are new to this spec from the previous version.

### PR DESCRIPTION
**Pull request by skeating**
Fixes #390 
Fixes #394
Fixes #398.
Fixes #400

This PR addresses all the issues that can be tackled without access to the uml diagram source

### CODE REVIEW

**Comment by bgoli** [comment]
Greetings and thanks @skeating! Added follow-up PR with updated UML, happy to finalize the document for release once these are merged.

File: sbml-level-3/version-1/fbc/spec/main.tex
Comment (bgoli) [inline sbml-level-3/version-1/fbc/spec/main.tex:1]: Just to mess with everyone's formatting, ORCID URLS are formally [complete URL's](https://support.orcid.org/hc/en-us/articles/360006897674-Structure-of-the-ORCID-Identifier#:~:text=Expressing%20the%20ORCID%20iD,%3A%2F%2Forcid.org%2F%22.). However, I like this macro so I would suggest leaving them as is.

File: sbml-level-3/version-1/fbc/spec/intro.tex
Comment (bgoli) [inline sbml-level-3/version-1/fbc/spec/intro.tex:1]: I generally remove the red from the release versions as I find i visually disturbing but please leave this in for future red 👍

File: sbml-level-3/version-1/fbc/spec/examples/ex-v3-userconstraints.txt
Comment (bgoli) [inline sbml-level-3/version-1/fbc/spec/examples/ex-v3-userconstraints.txt:1]: Looks good. I really like your example of not using a NaN for a parameter without predefined value (artificial variable).

**Review by bgoli — COMMENTED** [review]
I jumped in uninvited, but I approve!

File: sbml-level-3/version-1/fbc/spec/intro.tex
Comment (skeating) [reply sbml-level-3/version-1/fbc/spec/intro.tex:1]: yes fine by me

**Pull request merged**